src/MyEventEmitter.js

```diff
@@ -1,12 +1,97 @@
 'use strict';
 
-export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+class MyEventEmitter {
+  constructor() {
+    this.events = {};
+  }
+
+  on(eventName, callback) {
```

Comment:
let's check what on method should return https://nodejs.org/api/events.html#emitteroneventname-listener